android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/protocol/MobileWalletAdapterConfig.java

```diff
@@ -25,10 +28,12 @@ public class MobileWalletAdapterConfig {
     public final Object[] supportedTransactionVersions;
 
     public MobileWalletAdapterConfig(boolean supportsSignAndSendTransactions,
+                                     @IntRange(from = 0) long noConnectionWarningTimeoutMillis,
```

Comment:
Can we append this, rather than add it to the middle? Existing consumers of `walletlib` will see a compilation error, and it'll be easier to interpret when appended. Otherwise, valid values of  `maxTransactionsPerSigningRequest` would now be used for `noConnectionWarningTimeoutMillis`, moving the error to a later parameter that will be harder to interpret.